api/serverV2_test.go

```diff
@@ -29,7 +29,11 @@ func TestServerV2Start(t *testing.T) {
 	})
 	require.NoError(err)
 
-	svr.core.chainListener = nil
+	coreService, ok := svr.core.(*coreService)
+	if !ok {
+		require.Error(err)
+	}
+	coreService.chainListener = nil
```

Comment:
This is a weird test, which should not happen. Panic in the coreService if possible. Add a TODO

Reply:
yes